### PR TITLE
this.apple: CPU and Apple IIe Card identification

### PR DESCRIPTION
**Pull request by mgcaret**
Added a couple features.  Small reorg of code to accommodate some relative branches that went out of range with my additions.

### CODE REVIEW

File: desk.acc/this.apple.s [hunk 1]
```diff
@@ -251,6 +251,9 @@ str_iie:
 str_iie_enhanced:
         PASCAL_STRING "Apple IIe (enhanced)"
 
+str_iie_card:
+        PASCAL_STRING "Apple IIe Card"
```
Comment (inexorabletash) [inline desk.acc/this.apple.s]: But what shall we use for a bitmap? ;-)

(Seriously... I can do up a bitmap of the card easily enough...)

Comment (mgcaret) [reply desk.acc/this.apple.s]: I was just going to take some "inspiration" from the IIe Card app. :-)

**Comment by mgcaret** [comment]
Screen shot of CPU identification:

cpu

**Comment by mgcaret** [comment]
Screenshot of CPU identification:

cpu2

**Comment by mgcaret** [comment]
Additionally... I admit not reading the coding style carefully enough to see "all code is 6502 not 65C02"...

That being said, in the case of my PR we can be confident in the few cases where I inserted bra instead of jmp that it won't be a problem and it saves a few bytes.  Feel free to amend them to jmps if you disagree.

**Comment by JohnMBrooks** [comment]
> Long term I don't know if we need to keep anything '02 pure - how many systems supported DHR without a new CPU?)

The original Apple //e (unenhanced) have DHR and a 6502 CPU (I use one).

Keeping the code 6502 pure is also good if you want to have a hires or text version for II/II+/clones in the future.

File: desk.acc/this.apple.s (hunk 1 above)
Comment (mgcaret) [reply desk.acc/this.apple.s]: I made a bitmap, but it causes the end of the DA to go past $13FF...  is that a hard limit?

Comment (mgcaret) [reply desk.acc/this.apple.s]: Ahh it looks like we can go up to $1CFF unless I am mistaken.

Comment (mgcaret) [reply desk.acc/this.apple.s]: It came out better than I expected for a first try drawing in 1s and 0s in my text editor...

card_bmap

Comment (inexorabletash) [reply desk.acc/this.apple.s]: If README.md is still correct, the window/icon tables start at $1B00 so $1AFF would be the hard limit in AUX. 

(The DA right now replicates itself between main and aux for easy bank switching, but that doesn't need to be maintained.)

Comment (inexorabletash) [reply desk.acc/this.apple.s]: (and the current $13FF was a copy/paste from other DAs that needed to keep space free for file bufffers, totally arbitrary)

**Comment by inexorabletash** [comment]
(we should sneak in an easter egg where OA-CA-clicking the icon cycles the detected system for demo/screenshot purposes)

**Comment by inexorabletash** [comment]
Will you make further additions to this PR or should I merge it and we can iterate?

File: desk.acc/this.apple.s (hunk 1 above)
Comment (mgcaret) [reply desk.acc/this.apple.s]: Funny, I read that right and changed my .assert bounds check to $1B00, but somehow wrote $1CFF in my comment.  Brains are weird.

I'll see what the icon looks like with 2px verticals and some other tweaks.

**Comment by mgcaret** [comment]
I have a commit removing the BRA in favor of JMP and including the IIe Card icon.

I will edit the PR shortly.

**Comment by inexorabletash** [comment]
Awesome submission, thank you!

**Pull request merged**